src/core_plugins/kibana/public/kibana.js

```diff
@@ -44,3 +45,5 @@ chrome
     moment.tz.setDefault(tz);
   }
 });
+
+modules.get('kibana').run(Notifier.run);
```

Comment:
I like this approach much better, but I think `Notifier.run` should probably be `Notifier.pullMessagesFromUrl` or something